scipy/fft/_realtransforms.py

```diff
@@ -1,32 +1,63 @@
 import scipy.fftpack as _fftpack
+from ._basic import _dispatch
+from scipy._lib.uarray import Dispatchable
+import numpy as np
+import functools
 
 __all__ = ['dct', 'idct', 'dst', 'idst', 'dctn', 'idctn', 'dstn', 'idstn']
 
-def _doc_wrap(transform_func, new_func):
-    doc = transform_func.__doc__ or ''
-    new_func.__doc__ = doc.replace('fftpack', 'fft')
-    new_func.__name__ = transform_func.__name__
-    return new_func
+def _inherit_doc(fftpack_fun):
+    def inner(fft_fun):
+        doc = fftpack_fun.__doc__ or ''
+        fft_fun.__doc__ = doc.replace('fftpack', 'fft')
+        return fft_fun
```

Comment:
Can you just use:

https://github.com/scipy/scipy/blob/master/scipy/misc/doccer.py#L20

If not, this code should probably live in that module

Reply:
I need this specifically to replace `scipy.fftpack` with `scipy.fft` in the documentation examples.

Reply:
Ahh okay. Think it's worth moving (and maybe calling `replacer`?) in the same place as all the other docstring manipulation routines?

Reply:
Okay, that sounds reasonable.

Reply:
Done, called it `doc_replacer` as I thought that was more explicit.